docs/04-operation-guides/troubleshooting/evnt-01-eventing-troubleshooting.md

```diff
@@ -1,44 +1,125 @@
-   ```
+1. Check if sending an event returns an HTTP Status Code other than 2xx.
+
+    - If the HTTP status code is 4xx, check if you are sending the events in correct formats. Eventing supports two event formats (legacy and cloud events); see the Eventing [tutorials](../../03-tutorials/00-eventing) for more information.
+
+    -  If the HTTP status code is 5xx, check the logs from the Eventing publisher proxy Pod for any errors. To fetch these logs, run this command:
    
-   If the event dispatch log `"message":"event dispatched"` is not present for NATS backend, then the issue could lie with how the event was published.
\ No newline at end of file
+        ```bash
+        kubectl -n kyma-system logs -l app.kubernetes.io/instance=eventing,app.kubernetes.io/name=eventing-publisher-proxy
+        ```
+
+    - If the HTTP status code is 2xx, verify if you are sending the event on same event type as defined in the subscription.
+  
+        ```bash
+        kubectl -n {NAMESPACE} get subscriptions.eventing.kyma-project.io {NAME} -o jsonpath='{.spec.filter.filters}'
+        ```
+
+### Step 4: Check the Eventing Controller logs.
+
+1. Check the logs from the Eventing Controller Pod for any errors and to verify that the event is dispatched.
+To fetch these logs, run this command:
+
+    ```bash
+    kubectl -n kyma-system logs -l app.kubernetes.io/instance=eventing,app.kubernetes.io/name=controller
+    ```
+
+2. Check for any error messages in the logs. If the event dispatch log `"message":"event dispatched"` is not present for NATS backend, the issue could be one of the following:
+
+   - The subscriber (the sink) is not reachable or the subscriber cannot process the event. Check the logs of the subscriber instance.
+
+   - The event was published in a wrong format.
+
+   - Eventing Controller cannot connect to NATS Server.
+
+### Step 5: Check if the Subscription sink is healthy.
+
+1. Check whether the workload URL defined in the Subscription sink is correct and healthy to receive events. To get the sink from the Subscription, run this command:
+
+    ```bash
+    kubectl -n {NAMESPACE} get subscriptions.eventing.kyma-project.io {NAME} -o jsonpath='{.spec.sink}'
+    ```
+
+2. To check the health of the sink, run the following commands:
+
+    ```bash
+    kubectl -n default run --image=curlimages/curl --restart=Never sink-test-tmp -- curl --head {SINK_URL}
+    kubectl -n default logs sink-test-tmp 
+    kubectl -n default delete pod sink-test-tmp
+    ```
+
+    If the returned HTTP status code is not 2xx, check the logs of the subscriber instance.
+
+### Step 6: Check NATS JetStream status
+
+1. Check the health of NATS Pods. Run the command:
+
+    ```bash
+    kubectl -n kyma-system get pods -l nats_cluster=eventing-nats
+    ```
+
+2. Check if the stream and consumers exist in NATS JetStream by following the [JetStream troubleshooting guide](evnt-02-jetstream-troubleshooting.md).
+
+
```

Comment:
What do we suggest if all checks gave no clear result? Or are we sure that these diagnostic steps will uncover all potential issues?

Reply:
That sounds fine, thank you!